fixtures/blizzard/hot-box-rgbw.json

```diff
@@ -0,0 +1,239 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Hot Box RGBW",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Ken Harris"],
+    "createDate": "2022-04-06",
+    "lastModifyDate": "2022-04-06"
+  },
+  "comment": "The documentation splits 8-ch and 9-ch modes into \"A\" and \"B\" variants, where the only difference is the max strobe speed.  For OFL, I've encoded only a single variant of each, and used keywords \"slow\"/\"fast\" rather than absolute frequencies.",
+  "links": {
+    "manual": [
+      "https://api.blizzard.lighting/public/Products/hotbox%20rgbw/downloads/hotbox-manual-rev-b.pdf"
+    ],
+    "productPage": [
+      "https://www.blizzardpro.com/products/hotbox-rgbw"
+    ]
+  },
+  "physical": {
+    "dimensions": [127, 102, 127],
+    "weight": 2.7,
+    "power": 47,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "7× 10W RGBW LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [26, 26]
+    }
+  },
+  "availableChannels": {
+    "Hue": {
+      "capability": {
+        "type": "Generic"
+      }
+    },
+    "Saturation": {
+      "capability": {
+        "type": "Generic"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "comment": "CHECKME: \"Value\" (of \"HSV\") in lighting is the same as intensity, right?"
```

Comment:
It probably is, but I'm not sure, as there is also "Hue/Saturation/Intensity", "Hue/Saturation/Brightness" and "Hue/Saturation/Lightness", and I don't know the difference between them. See also #505.

Reply:
Changed in [`5072772`](https://github.com/OpenLightingProject/open-fixture-library/pull/2527/commits/507277214dbc00ad7da0ff7eb1a22531ad1cebd0).